### PR TITLE
tools: add `pd-ctl` download guide

### PR DESCRIPTION
**Pull request by weekface**

### CODE REVIEW

File: v3.0/reference/tools/pd-control.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ PD Control 是 PD 的命令行工具，用于获取集群状态信息和调整
...
+
+| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
+|:---|:---|:---|:---|
+| `http://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz` (pd-ctl) | Linux | amd64 | `http://download.pingcap.org/tidb-{version}-linux-amd64.sha256` |
+
+> **注意：**
+>
+> 下载链接中的 `{version}` 为 TiDB 的版本号。例如，`v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
+
+
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
```

File: v3.0/reference/tools/pd-control.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ PD Control 是 PD 的命令行工具，用于获取集群状态信息和调整
...
+如需下载最新版本的 `pd-ctl`，直接下载 TiDB 安装包即可，因为 `pd-ctl` 包含在 TiDB 安装包中。
+
+| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
+|:---|:---|:---|:---|
+| `http://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz` (pd-ctl) | Linux | amd64 | `http://download.pingcap.org/tidb-{version}-linux-amd64.sha256` |
+
+> **注意：**
+>
+> 下载链接中的 `{version}` 为 TiDB 的版本号。例如，`v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
> 下载链接中的 `{version}` 为 TiDB 的版本号。例如 `v3.0.5` 版本的下载链接为 `http://download.pingcap.org/tidb-v3.0.5-linux-amd64.tar.gz`。也可以使用 `latest` 替代 `{version}` 来下载最新的未发布版本。
```

**Comment by TomShawn** [comment]
Does this change apply to other versions? If so, please update the affected versions, thanks.

**Comment by weekface** [comment]
I opened a new PR: https://github.com/pingcap/docs-cn/pull/2095, closing

**Pull request closed without merging**